### PR TITLE
Prac 05 

### PR DESCRIPTION
**Pull request by shivprasad150204**
please review this @JiajunJiang-Oscar

### CODE REVIEW

File: prac_05/state_names.py
Comment (JiajunJiang-Oscar) [inline prac_05/state_names.py:1]: Good enough, but I think maybe can delete the "TODO" to means you finish this?

File: prac_05/hex_colours.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "aquamarine1": "#7fffd4", "aquamarine2": "#76eec6", "aquamarine4": "#458b74",
+    "azure1": "#f0ffff", "azure2": "#e0eeee", "azure3": "#c1cdcd", "azure4": "#838b8b",
+    "beige": "#f5f5dc", "bisque1": "#ffe4c4"
+}
+
+def get_colour_code(colour_name):
+    """Return the hexadecimal code for a given colour name."""
+    return COLOUR_CODES.get(colour_name, "Invalid colour name")
+
+def main():
```
Comment (JiajunJiang-Oscar) [inline prac_05/hex_colours.py:13]: no docstring, by the way, I think the main function should on the top of other functions

File: prac_05/wimbledon.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+"""
+CP1404/CP5632 Practical
+Wimbledon data-reading, processing, and displaying
+"""
+FILENAME = "wimbledon.csv"
+INDEX_COUNTRY = 1
+INDEX_CHAMPION = 2
+
+
+def main():
```
Comment (JiajunJiang-Oscar) [inline prac_05/wimbledon.py:10]: Should have 4 function

File: prac_05/hex_colours.py (hunk 1 above)
Comment (shivprasad150204) [reply prac_05/hex_colours.py:13]: good point

File: prac_05/wimbledon.py (hunk 2 above)
Comment (shivprasad150204) [reply prac_05/wimbledon.py:10]: thanks for reminding

File: prac_05/state_names.py
Comment (shivprasad150204) [reply prac_05/state_names.py:1]: I missed that, Thank you for pointing it out

**Pull request merged**